variables.tf

```diff
@@ -111,22 +111,22 @@ variable "load_balancer_sku" {
 }
 
 variable "managed_outbound_ip_count" {
-  default = null
+  default = 1
   type    = number
 }
 
 variable "outbound_ip_address_ids" {
   default = null
-  type    = list
+  type    = list(any)
 }
 
 variable "outbound_ip_prefix_ids" {
   default = null
```

Comment:
Should an empty list be the default?

Reply:
Nope, because if managed ip count then null, they mutually exclude eachother.

Reply:
Ok, I see. This goes for all of these then and "managed_outbound_ip_count" set to 1 is the default needed to have a valid config.